### PR TITLE
feat: add polygon mumbai support

### PR DESCRIPTION
**Pull request by luzzif**

### CODE REVIEW

File: packages/subgraph/package.json [hunk 1]
```diff
@@ -1,27 +1,30 @@
...
+        "@carrot-kpi/contracts": "^0.9.0",
+        "@graphprotocol/graph-cli": "0.60.0",
         "@graphprotocol/graph-ts": "0.31.0",
         "eslint-config-custom": "*",
-        "matchstick-as": "0.5.2",
+        "matchstick-as": "0.6.0",
         "rimraf": "^5.0.5",
-        "ts-node": "^10.9.1"
+        "ts-node": "^10.9.1",
+        "tsconfig": "*"
```
Comment (guerrap) [inline packages/subgraph/package.json:28]: Is this desired? To have any version?

File: packages/subgraph/subgraph.yaml [hunk 2]
```diff
@@ -7,11 +7,11 @@ features:
 dataSources:
   - kind: ethereum/contract
     name: Factory
-    network: gnosis
+    network: mumbai
```
Comment (guerrap) [inline packages/subgraph/subgraph.yaml:10]: Why did gnosis get replaced here?

File: packages/ui/src/utils/components.tsx [hunk 3]
```diff
@@ -12,8 +12,9 @@ if (theme) {
 
 const twMergeWithConfig = extendTailwindMerge({
     prefix: "cui-",
-    theme,
-    classGroups,
+    extend: {
```
Comment (guerrap) [inline packages/ui/src/utils/components.tsx:15]: Is theme not needed anymore?

File: packages/subgraph/package.json (hunk 1 above)
Comment (luzzif) [reply packages/subgraph/package.json:28]: `tsconfig` is a local dependency internal to the monorepo, so with `yarn` that's how you specify it.

File: packages/subgraph/subgraph.yaml (hunk 2 above)
Comment (luzzif) [reply packages/subgraph/subgraph.yaml:10]: That's a side effect of how the TheGraph's library handles the `--network` parameter passed when deploying a subgraph. It basically replaces the `network` and `address` fields in the `subgraph.yaml` file in place. The end result is that each time you do a deployment on a different network this file will be updated. So it's expected.

File: packages/ui/src/utils/components.tsx (hunk 3 above)
Comment (luzzif) [reply packages/ui/src/utils/components.tsx:15]: I tried to start up the frontend locally and didn't notice any difference really. I'm also not noticing any difference in the UI preview above.

**Review by guerrap — APPROVED** [review]

**Pull request merged**